_rules/device-motion-bf0de1.md

```diff
@@ -0,0 +1,327 @@
+---
+id: bf0de1
+name: No functions are exclusively operated through device motion
+rule_type: composite
+description: |
+  This rule checks that functionality that can be operated by device motion or user motion can also be operated by user interface components and responding to the motion can be disabled to prevent accidental actuation
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_rules:
+  - 5d9784
+  - de951c
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).
+
+## Expectation
+
+For each test target, the outcome of all of the following rules is passed:
+
+- [Motion can be replaced by user interface components](https://act-rules.github.io/rules/5d9784)
+- [Motion can be disabled](https://act-rules.github.io/rules/de951c)
+
+## Assumptions
+
+- This test assumes that motion is not [essential](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-essential) for the function

```

Comment:
and that _The motion is used to operate functionality through an accessibility supported interface;_  ? no?